x/mock/app.go

```diff
@@ -217,7 +217,7 @@ func GenTx(msgs []sdk.Msg, accnums []uint64, seq []uint64, priv ...crypto.PrivKe
 		Gas:    100000,
 	}
 
-	sigs := make([]auth.StdSignature, len(priv))
+	sigs := make([]sdk.Signature, len(priv))
```

Comment:
there's another one in `app_test.go` L33 in this same module, can we use `sdk.Signature` there too ?

Reply:
done